### PR TITLE
Update Address and Note character limit

### PR DESCRIPTION
**Pull request by darrentfy**

### CODE REVIEW

**Comment by darrentfy** [comment]
Also deleted the unused Email class

File: src/main/java/seedu/address/model/student/Address.java [hunk 1]
```diff
@@ -9,13 +9,14 @@
...
-    public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS = "Addresses should not be blank, "
+            + "and can have a maximum of 200 characters";
 
     /*
      * The first character of the address must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
-    public static final String VALIDATION_REGEX = "[^\\s].*";
+    public static final String VALIDATION_REGEX = "[^\\s].{0,199}";
```
Comment (SimWPEric) [inline src/main/java/seedu/address/model/student/Address.java:19]: Good update for upper bound !

File: src/main/java/seedu/address/model/student/Note.java [hunk 2]
```diff
@@ -9,8 +9,8 @@
 public class Note {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Note should be 200 characters or less";
-    public static final String VALIDATION_REGEX = ".{0,200}";
+            "Note should be 500 characters or less";
+    public static final String VALIDATION_REGEX = ".{0,500}";
```
Comment (SimWPEric) [inline src/main/java/seedu/address/model/student/Note.java:13]: Good change to accommodate new Notes column, as notes will probably be longer now.

**Review by SimWPEric — APPROVED** [review]
LGTM, good removal of redundant classes like `Email`.

**Pull request merged**